### DISCUSSION TITLE
A good Analytics Framework to integrate with?

### DISCUSSION THREAD

**Discussion by shrekuntu** [markfairbanks/tidypolars#151] [General]
I'm fairly new to Data Science fields. Started with pandas, then discovered R and tidyverse and I was hooked.
Searching for a python version of tidyverse, found your project, although it wasn't an easy find.

I need to build a business analytics dashboard and plotly dash looks like a best option of those i found. Wondering if dash would work with polars tibbles?

Hope tidypolars gains traction as main value proposition are the speed of polars and clean syntax close to tidyverse.
Good luck with your project and keep up the good work.

**Comment by markfairbanks** [comment]
Hey, how's it going?

> Wondering if dash would work with polars tibbles?

I don't have much experience with dash unfortunately. I'm guessing it wouldn't work with tibbles directly (but I could be wrong). There are a couple options that I'm guessing would work:

1. [Converting to a pandas DataFrame](https://github.com/markfairbanks/tidypolars#converting-tofrom-pandas-data-frames)
2. Converting to a dictionary (FYI - needs tidypolars v0.2.7):
  i. `your_df.to_dict()`: Convert a Tibble to a dictionary with Series as the values
  ii. `your_df.to_dict(as_series = False)`: Convert a Tibble to a dictionary with lists as the values

> Searching for a python version of tidyverse, found your project, although it wasn't an easy find.

Feel free to let more people know about tidypolars 😄 We just started advertising it a week or so ago. The project itself is only a couple months old and is now ready for general use.

> Good luck with your project and keep up the good work.

Thanks! If you run into any issues or other questions let me know! We've put a lot of work in, but I'm sure there are some small kinks left to figure out.
